.github/workflows/deploy.yml

```diff
@@ -0,0 +1,163 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - master
+
+  # Run actions on PRs, but only deploy on master
+  pull_request:
```

Comment:
Are you sure this doesn't deploy on PRs?

Reply:
Yes, the deploy steps are skipped when the ref is not master.

Reply:
Ah, yes sorry I missed that check.